AppCenterAnalytics/AppCenterAnalytics/Internals/Model/MSEventPropertiesInternal.h

```diff
@@ -0,0 +1,43 @@
+#import <Foundation/Foundation.h>
+
+#import "MSEventProperties.h"
+
+@class MSTypedProperty;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Typed event properties.
+ */
+@interface MSEventProperties () <NSCoding>
+
+/**
+ * String and date properties.
+ */
+@property (nonatomic) NSMutableDictionary<NSString *, MSTypedProperty *> *properties;
+
+/**
+ * Creates an instance of EventProperties with a string-string properties dictionary.
+ *
+ * @param properties A dictionary of properties.
```

Comment:
Add a line between `@param` and `@return`, please.

Reply:
good catch 👍